.github/workflows/helm-test.yaml

```
@@ -0,0 +1,79 @@
+#################################################################################
+# Copyright (c) 2022,2023 T-Systems International GmbH
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+
+name: "Lint and Test Charts"
+
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches:
+      - main
+    paths:
+      - 'SDFactory/**'
+      
+  pull_request:
+    branches:
+      - main
+jobs:
+    lint-test:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+              with:
+                fetch-depth: 0
+
+            - name: Kubernetes KinD Cluster
+              uses: container-tools/kind-action@v1
+        
+            - name: Build image
+              uses: docker/build-push-action@v3
+              with:
+                context: .
+                push: true
+                tags: kind-registry:5000/sd-factory:testing
+        
+            - name: Set up Helm
+              uses: azure/setup-helm@v3
+              with:
+                version: v3.9.3
+        
+            - uses: actions/setup-python@v4
+              with:
+                python-version: '3.9'
+                check-latest: true
+        
+            - name: Set up chart-testing
+              uses: helm/chart-testing-action@v2.3.1
+        
+            - name: Run chart-testing (list-changed)
+              id: list-changed
+              run: |
+                changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+                if [[ -n "$changed" ]]; then
+                   echo "::set-output name=changed::true"
+                fi
+            - name: Run chart-testing (lint)
+              run: ct lint --validate-maintainers=false --target-branch ${{ github.event.repository.default_branch }}
+        
+            - name: Run chart-testing (install)
+              run: ct install --charts charts/SDFactory --helm-extra-set-args "--set=image.tag=testing"
```

Can you try the following change?

```suggestion
              run: ct install --charts charts/SDFactory --helm-extra-set-args "--set=image.tag=testing --set=image.repository=sd-factory"
```